influxql/functions.go

```diff
@@ -211,29 +211,31 @@ func MapMean(itr Iterator) interface{} {
 
 	for _, k, v := itr.Next(); k != 0; _, k, v = itr.Next() {
 		out.Count++
-		out.Sum += v.(float64)
+		out.Mean += (v.(float64) - out.Mean) / float64(out.Count)
 	}
 	return out
 }
 
 type meanMapOutput struct {
 	Count int
```

Comment:
@toddboom i had a question about this... should `Count` be a `float64`?  everywhere it gets used (except for where it's calculated) it gets cast to `float64` and i don't know go well enough to know what is more idiomatic.